spec/acceptance/ios_stp_global_spec.rb

```diff
@@ -33,6 +33,7 @@
     pp = <<-EOS
     ios_stp_global { 'default':
       bridge_assurance => true,
```

Comment:
`bridge` also does not seem to be a spanning-tree configuration on the 4503

```
no spanning-tree ?
  backbonefast  Enable BackboneFast Feature
  etherchannel  Spanning tree etherchannel specific configuration
  extend        Spanning Tree 802.1t extensions
  logging       Enable Spanning tree logging
  loopguard     Spanning tree loopguard options
  mode          Spanning tree operating mode
  mst           Multiple spanning tree configuration
  pathcost      Spanning tree pathcost options
  portfast      Spanning tree portfast options
  transmit      STP transmit parameters
  uplinkfast    Enable UplinkFast Feature
  vlan          VLAN Switch Spanning Tree
```

Reply:
The existing test would've quietly failed, it may be worth taking this out of the acceptance test as it isn't guaranteed to run.